data/beam-pri.mesh

```diff
@@ -0,0 +1,128 @@
+MFEM mesh v1.0
```

Comment:
Do we need this file in addition to `beam-wedge.mesh`?

Reply:
No, this was an oversight.